pkg/auth/authProperties.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package auth
+
+import (
+	"path/filepath"
+
+	"github.com/galasa.dev/cli/pkg/files"
+	"github.com/galasa.dev/cli/pkg/galasaapi"
+	"github.com/galasa.dev/cli/pkg/props"
+	"github.com/galasa.dev/cli/pkg/utils"
+
+	galasaErrors "github.com/galasa.dev/cli/pkg/errors"
+)
+
+const (
+	CLIENT_ID_PROPERTY    = "GALASA_CLIENT_ID"
+	SECRET_PROPERTY       = "GALASA_SECRET"
+	ACCESS_TOKEN_PROPERTY = "GALASA_ACCESS_TOKEN"
+)
+
+// Gets authentication properties from the user's galasactl.properties file
+func GetAuthProperties(fileSystem files.FileSystem, galasaHome utils.GalasaHome) (galasaapi.AuthProperties, error) {
+    var err error = nil
+    authProperties := galasaapi.NewAuthProperties()
+
+    galasactlPropertiesFilePath := filepath.Join(galasaHome.GetNativeFolderPath(), "galasactl.properties")
+    galasactlProperties, err := props.ReadPropertiesFile(fileSystem, galasactlPropertiesFilePath)
+    if err == nil {
+        clientIdProperty := "GALASA_CLIENT_ID"
+        secretProperty := "GALASA_SECRET"
+        accessTokenProperty := "GALASA_ACCESS_TOKEN"
+
+        requiredAuthProperties := getAuthPropertiesList()
+        err = validateRequiredGalasactlProperties(requiredAuthProperties, galasactlProperties)
+
+        if err == nil {
+            authProperties.SetClientId(galasactlProperties[clientIdProperty])
```

Comment:
Or perhaps just use the symbolic constants here ?

Reply:
Thanks, forgot to change that! Addressed in https://github.com/galasa-dev/cli/pull/151/commits/de77ebc07ca7a859ab4cdce44c527f106d88f801